locust/stats.py

```diff
@@ -63,7 +64,13 @@ def clear_all(self):
         self.max_requests = None
         self.last_request_timestamp = None
         self.start_time = None
-        
+
+    def total_run_time(self):
+        try:
+            # return the time that the test has been running for.
+            self.run_time = int(time.time() - self.start_time)
+        except Exception as ex:
+            print(ex)
```

Comment:
is it useful to just print the exception and continue like this?
this exception doesn't seem like it could ever be raised here.